exercises/triangle/description.md

```diff
@@ -1,28 +1,20 @@
-The program should raise an error if the triangle cannot exist.
+Determine whether a triangle is equilateral, isoceles, or scalene.
```

Comment:
because this is the same line as https://github.com/exercism/x-common/blob/master/exercises/triangle/metadata.yml#L2 plus the fact that we prepend the blurb line to the README, this will cause this line to appear twice.

Thus, I would suggest it be removed from README.